src/components/common/Stars/Stars.js

```diff
@@ -118,6 +118,7 @@ function Stars(props) {
   }
 
   function getStars(i) {
+<<<<<<< HEAD
```

Comment:
hmm, to nie wiem o co chodzi, tutaj wyświetlają się jakby obie opcje

Reply:
poprawilem